pkg/koordlet/statesinformer/states_nodemetric.go

```diff
@@ -330,8 +330,8 @@ func (r *nodeMetricInformer) collectMetric() (*slov1alpha1.NodeMetricInfo, []*sl
 	}
 	for _, podMeta := range podsMeta {
 		podMetric := r.collectPodMetric(podMeta, podQueryParam)
-		r.fillExtensionMap(podMetric, podMeta.Pod)
 		if podMetric != nil {
+			r.fillExtensionMap(podMetric, podMeta.Pod)
```

Comment:
could you please add a unit test

Reply:
I could, but it's an empty function. 😂😂😂